build_infrastructure/vars/main.yml

```diff
@@ -0,0 +1,25 @@
+terraform_local_dir: "{{ terraform_base_dir }}/{{ scenario_name }}"
+terraform_remote_dir: "{{ tmp_dir }}/terraform"
+key_path: "{{ tmp_dir }}/{{ key_name }}"
+
+backend_config:
+  key: "terraform_state/{{scenario_name}}"
+  endpoint: "obs.eu-de.otc.t-systems.com"
+  bucket: "obs-csm"
+  region: "eu-de"
+
+username: "{{ lookup('env', 'OS_USERNAME') }}"
+password: "{{ lookup('env', 'OS_PASSWORD') }}"
+
+auth_url: https://iam.eu-de.otc.t-systems.com/v3
+user_domain_name: OTC00000000001000000448
```

Comment:
Hard link to 448 is it ok?

Reply:
i have't got any idea... what about you?

Reply:
maybe will we use lookup for project_name and domain name?

Reply:
i think only username and password are secure date